acceptance/tests/options/color.rb

```diff
@@ -0,0 +1,15 @@
+# This test is intended to ensure with --debug and --color, facter sends escape sequences to colorize the output
+test_name "C864545: --debug and --color command-line options should print DEBUG messages with color escape sequences" do
+  tag 'risk:high'
```

Comment:
+1 to @kurtwall 's risk assessment comment. This seems both low severity to the user and low probability of regression. I would rate it as 'risk:low'